<launch>

  <!-- Trying to get camera feed from Jackal -->
  <!-- grab camera image feed and publish -->
  <node name="image_view" pkg="image_view" type="image_view_node" output="screen" >
    <param name="pixel_format" value="yuyv" />

    <param name="image" value="camera/image_raw" />
  </node>

</launch>
